lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-ansible_tower-automation_manager-playbook.rb

```diff
@@ -0,0 +1,5 @@
+module MiqAeMethodService
+  class MiqAeServiceManageIQ_Providers_AnsibleTower_AutomationManager_Playbook <
+      MiqAeServiceConfigurationScriptPayload
```

Comment:
I'm not a fan of breaking the line...just keep it one line (We turned off the 120 char rule :P )